dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -38,6 +37,115 @@ TypeInfo TypeFromNumpyStr(const std::string &format) {
   DALI_FAIL("Unknown Numpy type string");
 }
 
+inline void SkipSpace(const char*& ptr) {
```

Comment:
```suggestion
inline void SkipSpaces(const char*& ptr) {
```
Small suggestion, your call